mne_bids/dig.py

```diff
@@ -228,10 +228,6 @@ def _write_coordsystem_json(*, raw, unit, hpi_coord_system,
     if dig is None:
         dig = []
     coords = _extract_landmarks(dig)
-    hpi = {d['ident']: d for d in dig if d['kind'] == FIFF.FIFFV_POINT_HPI}
-    if hpi:
-        for ident in hpi.keys():
-            coords['coil%d' % ident] = hpi[ident]['r'].tolist()
```

Comment:
thanks for removing this unnecessary code :) 

can you perhaps move line 230 `coords = _extract_landmarks(dig)`

right before the if meg, elif eeg, ... logic clause (below the comment in line 255)?

I feel like that would make the code more readable, because `coords` isn't needed until then.

Reply:
Yup, makes sense.